root/components/Table.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {useTable} from 'react-table';
+
+import loopParity from '../utility/loopParity';
+
+const Table = (({columns, data}: any) => {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
+    columns,
+    data,
+  });
+
+  /* eslint-disable react/jsx-key */
+  return (
+    <table {...getTableProps({className: 'tbl'})}>
+      <thead>
+        {headerGroups.map(headerGroup => (
+          <tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <th
+                {...column.getHeaderProps({
+                  className: column.className,
+                })}
+              >
+                {column.render('Header')}
+              </th>
+            ))}
+          </tr>
+        ))}
+      </thead>
+      <tbody {...getTableBodyProps()}>
+        {rows.map(
+          (row, i) => prepareRow(row) || (
```

Comment:
When I read this, I assumed `prepareRow` was some kind of default renderer, with `||` providing a display fallback if that produced null or undefined. But it appears that `prepareRow` has no return value and is simply mandatory to call, with `||` being used to sequence things.

The code would be much clearer if written as
```JS
(row, i) => {
  prepareRow(row);
  return (
    <tr>
      {/* ... */}
    </tr>
  );
}
```

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1246/commits/6ee5d3d424b1dda8aecefe6be1b9f7d2ac4092e1#diff-dfa9d33d3b7fc4efebab124f77237409R15-R64